### PR TITLE
trello 1.1 

### PR DESCRIPTION
**Pull request by sathya0**
interim commit

### CODE REVIEW

**Review by valean27 — DISMISSED** [review]

File: trello/1.1/modules/ROOT/pages/trello-connector-reference.adoc [hunk 1]
```diff
@@ -23713,6 +23787,83 @@ Update Webhooks Id Model By Id Webhook. This operation makes an HTTP PUT request
...
+
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use | | x
+| Organization ID a| String |  ID of the organization |  |
+| Since timestamp a| String |  Timestamp of since when to query new or updated cards |  |
```
Comment (dhirsch1) [inline trello/1.1/modules/ROOT/pages/trello-connector-reference.adoc]: I suggest: Timestamp for querying new cards or updating existing cards

File: trello/1.1/modules/ROOT/pages/trello-connector-reference.adoc [hunk 2]
```diff
@@ -23713,6 +23787,83 @@ Update Webhooks Id Model By Id Webhook. This operation makes an HTTP PUT request
...
+| Config Ref a| ConfigurationProvider |  Name of the configuration used to execute this component |  |
+| Primary Node Only a| Boolean |  Whether this source should only be executed on the primary node when running in Cluster |  |
```
Comment (dhirsch1) [inline trello/1.1/modules/ROOT/pages/trello-connector-reference.adoc]: There is a description for this in the Reference spreadsheet: https://docs.google.com/spreadsheets/d/1FzjhPtRWNHmaNK_8JbXbkbblZiul6zELQ2I4Hys0_UI/edit#gid=0

Please use it for all occurrences of this field.

File: trello/1.1/modules/ROOT/pages/trello-connector-reference.adoc [hunk 3]
```diff
@@ -23917,3 +24070,25 @@ File store repeatable streams require buffering, and there are different bufferi
...
+|===
+
+[[RedeliveryPolicyMessageIdentifier]]
+=== Redelivery Policy Message Identifier
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|===
+| Field | Type | Description | Default Value | Required
+| Use Secure Hash a| Boolean | Whether to use a secure hash algorithm to identify a redelivered message |  |
+| Id Expression a| String | Defines one or more expressions to use to determine when a message has been redelivered. This property may only be set if useSecureHash is false. |  |
```
Comment (dhirsch1) [inline trello/1.1/modules/ROOT/pages/trello-connector-reference.adoc]: may > can

**Review by dhirsch1 — DISMISSED** [review]
I had some nit-picky comments but you don't have to address them since this is a REST SDK connector. I see that I forgot to put the version in the Reference header, so it would be good if you did that. Also, we can start removing Mule 4 from the titles, but if you are busy, this can wait.

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**